pkg/cloudprovider/providers/gce/gce_metrics.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gce
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var gceMetricMap = map[string]*prometheus.HistogramVec{
+	"gce_instance_list": prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "gce_instance_list",
+			Help:    "Latency of instance listing calls in ms",
+			Buckets: prometheus.ExponentialBuckets(100, 2, 10),
+		},
+		[]string{"kube_namespace"},
```

Comment:
According to the [instrumentation guide](https://github.com/kubernetes/community/blob/master/contributors/devel/instrumentation.md), this should be just `namespace`.